src/components/form-control/index.tsx

```diff
@@ -130,7 +134,55 @@ const FormControl: FC<Props> = ({
               }
             >
               {localization.publish}
-            </SC.PublishButton>
+            </SC.StatusButton>
+          )}
+          {!published && (
+            <SC.StatusButton
+              $active={lastPatchedResponse?.status === ConceptStatus.HOERING}
+              disabled={
+                validationError ||
+                (!!concept.revisjonAv && !concept.revisjonAvSistPublisert)
+              }
+              onClick={() =>
+                patchConceptFromForm(
+                  {
+                    status: ConceptStatus.HOERING,
+                    ...(concept.versjonsnr?.major === 0 &&
+                      concept.versjonsnr?.minor === 0 &&
+                      concept.versjonsnr?.patch === 1 && {
+                        versjonsnr: { major: 1, minor: 0, patch: 0 }
+                      })
+                  },
+                  { concept, dispatch, lastPatchedResponse, isSaving }
+                )
+              }
+            >
+              {localization.setToHearing}
```

Comment:
Hearing blir vel litt feil for høring i denne settingen, kanskje evaluation, assessment eller consideration?